build/automation/var/platform-texas/v1/account-live-k8s-nonprod.mk

```diff
@@ -1,5 +1,8 @@
 AWS_ACCOUNT_ID := $(AWS_ACCOUNT_ID_NONPROD)
 AWS_ACCOUNT_NAME := nonprod
+AWS_CERTIFICATE = c0718115-4e22-4f48-a4aa-8c16ea86c5e6
+
+WAF_ACL_ID = dfae6ec3-aa05-428f-a022-5fd85f646009
```

Comment:
Is this the Texas WAF? If so, that's the right place for this config.

Reply:
Sure